api.rst

```diff
@@ -365,7 +365,7 @@ The keys of the object match the parameter names. Note that PostgreSQL converts
 
 PostgreSQL has four procedural languages that are part of the core distribution: PL/pgSQL, PL/Tcl, PL/Perl, and PL/Python. There are many other procedural languages distributed as additional extensions. Also, plain SQL can be used to write functions (as shown in the example above).
 
-By default, a function is to be executed with the privileges of the user that calls it. This means, that the user has to have all permissions to do all operations the procedure performs. But if the function was defined with `SECURITY DEFINER` options, only one permission check will take place – the permission to call the function. See [PostgreSQL documentation](https://www.postgresql.org/docs/current/static/sql-createfunction.html) for more details.
+By default, a function is to be executed with the privileges of the user that calls it. This means, that the user has to have all permissions to do all operations the procedure performs. But if the function was defined with :code:`SECURITY DEFINER` options, only one permission check will take place – the permission to call the function. See `PostgreSQL documentation <https://www.postgresql.org/docs/current/static/sql-createfunction.html>` for more details.
```

Comment:
One last little thing, links end with a trailing underscore. It's weird, I know...

```rst
`title <url>`_
```

http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets